dear_petition/petition/constants.py

```diff
@@ -99,14 +99,22 @@
     "PROBATION OTHER": "PO",
     "PROBATION REVOKED": "PR",
     "REMANDED TO DISTRICT COURT": "RM",
-    SUPERSEDING_INDICTMENT_OR_PROCESS: "SI",
+    DISP_SUPERSEDING_INDICTMENT: "SI",
```

Comment:
During development, I had made another one of these dispositions a constant (but later removed it because I ended up not needing it).  

When there were two (and there could be two or more again at some future point), it helped with code readability (IMHO) to prefix them with what they are (a disposition). 

Plus, if you don't prefix them with what they are, then sometimes there will be conflicts with other constants that happen to have the same value.  For example I also made a constant for the verdict "GUILTY" `VERDICT_GUILTY = "GUILTY"`.  I could have just called it `GUILTY = "GUILTY"`.  And then another developer might come along and want to use a constant for the plea "GUILTY".  But they really shouldn't be using the same constant for both verdict and plea because one could potentially change without the other.  But if it is prefixed, hopefully the second developer would realize how it is used and make the constant `PLEA_GUILTY = "GUILTY"`.